projects/bc-java-wycheproof/project.yaml

```diff
@@ -8,3 +8,5 @@ sanitizers:
   - "none"
 file_github_issue: false
 disabled: true
+vendor_ccs:
+- hlin@code-intelligence.com
```

Comment:
This has nothing to do with code intelligence, please remove this.

Reply:
I deleted 12 projects which are not onboarded by CI, sorry for the mistake!

Reply:
No worries. My comment came across harsher than I intended.